recipes/pykml/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pykml" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "6efaa4c31ddeefbbdd256a278175a38aea556b08e752f340098c95229a2df66a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - pykml = pykml:main
+    - kml2pykml = pykml.factory:kml2pykml
+    - csv2kml = pykml.util:csv2kml
+    - validate_kml = pykml.parser:validate_kml
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - lxml >=2.2.6
+
+  run:
+    - python
+    - setuptools
+    - lxml >=2.2.6
+
+test:
```

Comment:
Please add a `commands:` section to `test` to test all of the console scripts.  See:

http://conda.pydata.org/docs/building/meta-yaml.html#test-commands